_observing-your-data/metricsanalytics.md

```diff
@@ -0,0 +1,205 @@
+
+You can see metrics from remote OpenSearch clusters by using the **Metrics analytics** tool. Select the database icon on the upper-right toolbar and choose the cluster you want from **DATA SOURCES** dropdown menu, as shown in the following image. You can switch from a local cluster to a remote cluster.
+
+<img src="{{site.url}}{{site.baseurl}}/images/metrics/remote-cluster-selection.png" alt="Switching clusters using the Metrics analytics tool" width="700"/>
+
+You can also view metrics visualizations from other sources together with local metrics visualizations. From the **DATA SOURCES** dropdown menu, choose the remote metrics visualization to add it to the group of visualizations already shown on the dashboard. An example dashboard is shown in the following image.
+
+<img src="{site.url}}{{site.baseurl}}/images/metrics/otel-metrics-remote-cluster-selection.png" alt="Metrics dashboard" width="700"/>
+
+To learn about multi-cluster support for data sources, see [Enable OpenSearch Dashboards to support multiple OpenSearch clusters](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1388).
+
+## Creating visualizations based on custom metrics
+
+You can create visuals from the metrics data collected by your OpenSearch cluster, including Prometheus metrics and custom metrics.
+
+To create these visualizations, follow these steps:
+
+1. From the OpenSearch Dashboards main menu, navigate to **Observability** > **Metrics** > **Available Metrics**.
+2. Choose the metrics to add to your visualization and then select **Save**.
+3. When prompted for a **Custom operational dashboards/application**, choose one of the listed options. You can edit the predefined name values in the **Metric Name** field.
+4. Select **Save** to save your visualization. An example visualization is shown in the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/metrics/metrics2.png" alt="Metrics analytics dashboard with visualizations" width="700"/>
+
+## Defining PPL queries for Prometheus metrics
+
+You can define [Piped Processing Language (PPL)]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/index/) queries to interact with metrics collected by Prometheus. The following is an example PPL query for a Prometheus metric:
+
+```
+source = my_prometheus.prometheus_http_requests_total | stats avg(@value) by span(@timestamp,15s), handler, code
+```
+{% include copy-curl.html %}
+
+### Creating a custom visualization based on the PPL query
+
+To create a custom visualization based on the PPL query, follow these steps:
+
+1. From the **Events Analytics** window, enter your PPL query and select **Refresh**. This will display the **Explorer** page.
+2. On the **Explorer** page, select **Save**.
+3. When prompted to choose a **Custom Operational Dashboards/Application**, select one of the listed options. Optionally, you can edit the predefined name values under the **Metric Name** fields and can choose to save the visualization as a metric.
```

Comment:
```suggestion
3. When prompted to choose a **Custom Operational Dashboards/Application**, select one of the listed options. Optionally, you can edit the predefined name values in the **Metric Name** fields and can choose to save the visualization as a metric.
```

Reply:
@YANG-DB Is Events Analytics now Explorer? I don't see a tool named "Events Analytics" listed in Observability.